### ISSUE TITLE
Bigger item hitboxes 

### ISSUE THREAD

**Issue by Warfork** [TeamForbiddenLLC/warfork-qfusion#69]
https://www.twitch.tv/vjtengen/clip/CrepuscularCleanPorcupineUWot?filter=clips&range=all&sort=time

Suggested by @tontoepfer#2168 Would need to be discussed further with vic\mike.

**Comment by Warfork** [comment]
> farmz0r 
> they are playble but at least to me it would be a lot more fun if they were bigger (like 1.5times size). repeadetly takes the fun outa games/makes me rage hard to miss an item while pretty much walking into it's spot, but that spot is actually only 5mm big and smaller than the drawn item. the hexagon hitboxes don't exactly help either. and it's ridiculous that you can effortlessly jump over items (which are actually flying in the air already) with an ordinairy jump. not exactly run-friendly, in a game that sets itself apart with its ability to make runs. wsw doesn't capitalize its strength and uniqueness properly.
> (and while the smallness can be argued to add some sort of skill/expertise dimension (in a way that you have to prepare and execute a route kinda precisely), defusing the item-smallness-factor a bit definitely won't make movement skill/precission obsolete/matter any less. that's just ridiculous, there will still be plenty of strafing/tricking/routing to get around the map quickly. as indicated above, it's rather the opposite - increasing sizes means reducing the amount of situations where movement/racing skills/getting there first cannot be utilized/turned into getting the item. that is because precise routings (like atm often needed) are easily disturbed/unavaillable (if.e. your exact start position for a run often gets determined by how a fight turns out, you eat dmg and get pushed in the air - even if just little, you have to insert some dodgyness,...  lota factors impacting where you can land and with what speed). 
>  tl;dr: imo what sets wsw/wf apart and is proably the most enjoyable aspect of it (at least to me) is runing around. we shouldn't make it as hard as possible to pick up items while doing that, or even turn it into a gamble by puting items 0.5mm above the ground and give them 2 nanometer radius. especially since you are actually supposed to pick up stuff while going backwards/sidewards in duel. why on earth would i want to jump over ra. (edited)
>

**Comment by Warfork** [comment]
> msh
> i think that's why items became smaller
> 
> that could be fixed if distance takes priority or something though
> 
> SD2 
> Pickup radius should either be made bigger or be accurately represented. (edited)
> 
> mike (Cocaine Diesel) 
> you should be able to jump over items because it adds more skill to the game and you should be punished for being unskilled
> 
> because that's what arena fps is all about
> 
> farmz0r
> yeah no it doesn't
> 
> Oizo
> Mike is right.
> 
> farmz0r 
> i like how a ca-only player is talking about itemsize. it is such a short sighted pov anyway. this  actually prevents opportunities for skillfull  plays, far more than it rewards skills. it's always a question about what sort of skills one wants to reward anyway, movement skills, orientation skills, tactical play, aiming,... . you kinda reward tactics and orientation with keeping them small, while allowing more on the fly pickups would allow racing guys to to not abbort movement more often and benefit more from their quickness (basically rewarding racing skills instead). (edited)
> 
> SD2
> Well at the very least the items should more accurately represent the radius
> 
> msh 
> they're boxes
